### DISCUSSION TITLE
Cloudflare Web Analytics

### DISCUSSION THREAD

**Discussion by mu88** [EFForg/privacybadger#2831] [Q&A] [answered]
Hey guys 👋🏻

What are your thoughts about Cloudflare Web Analytics (https://cloudflareinsights.com)? From my (rather poor) understanding, I'd consider this a tracker as well.

Thanks in advance!

**Comment by ghostwords** [comment, marked as answer]
Hello!

Just to be clear, there is no manually-assembled block list in Privacy Badger; [Privacy Badger automatically learns](https://privacybadger.org/#How-does-Privacy-Badger-work) to block trackers by observing their behavior.

At first glance, it sounds like Privacy Badger would not learn to block Cloudflare Web Analytics because Cloudflare promises not to track individual user behavior:

>Cloudflare Web Analytics does not use any client-side state, such as cookies or localStorage, to collect usage metrics. We also don’t “fingerprint” individuals via their IP address, User Agent string, or any other data for the purpose of displaying analytics.

-- https://www.cloudflare.com/web-analytics/

**Reply by mu88** [reply]
Okay, thank you for the clarification! You know, it just seems a bit strange to me that Cloudflare offers a privacy-respecting analyzer for free. It's a bit like _if you don't pay for it, you are the product_

**Reply by ghostwords** [reply]
I think buit-in analytics make Cloudflare's product suite more attractive to all sorts of customers. And once you sign up for Cloudflare services, it probably becomes much more likely you'll become a paying customer at some point.

Offering a privacy-respecting service could also be a way to differentiate and market your products.
